### ISSUE TITLE
Simplify API for attaching local RawContacts to an Account

### ISSUE THREAD

**Issue by vestrel00** [vestrel00/contacts-android#152]
While working on #147 in preparation for #144 and #145, I noticed that the `AccountsRawContactsAssociationsUpdate` can be simplified to adhere to the pattern followed by the other APIs in the library.

The API can be simplified to,

```kotlin
val result = accountsLocalRawContactsUpdate
      .addToAccount(account)
      .localRawContacts(rawContacts)
      .commit()
```

**Comment by vestrel00** [comment]
This will be closed in scope of #147

**Comment by vestrel00** [comment]
Included in release; https://github.com/vestrel00/contacts-android/releases/tag/0.1.10
